uiowa/uiowa
# re-export admissions config based on recent changes

# To Test

- Checkout this branch.
- `ddev composer install && ddev blt frontend && ddev drush @admissions.local sql-drop && ddev drush @admissions.local cr && ddev blt ds --site=admissions.uiowa.edu`
- There should be no config import issues.
- `ddev drush @admissions.local uli` go to https://admissions.uiowa.ddev.site/admin/structure/taxonomy/manage/tags/overview/form-display and see that both image and tvi are represented.
